add 2026-07-16 doc (#110)
sky-ecosystem/executive-votes: 2026/executive-vote-2026-07-16-june-msc-staking-rewards-normalization.md

---

> title: Template - [Executive Vote] Monthly Settlement Cycle for June 2026, LSSKY-SKY Rewards Normalization, Complete RWA001-A Offboarding, Add Emergency Spells to the Chainlog, Whitelist Osero ALMProxy, Adjust Vault Parameters, Prime Agent Proxy Spells - July 16, 2026

Reviewer: Add Safe Harbor update here and elsewhere like previous docs

> summary: Execute the Monthly Settlement Cycle for June 2026 and the associated reconciliation transfers, normalize LSSKY-SKY staking rewards, complete the offboarding of RWA001-A, add emergency spells for STUSDS_MOM to the Chainlog, whitelist the new Osero DPAU ALMProxy on the LitePSM, adjust the Osero DC-IAM parameters, whitelist Prime spells for Spark, Grove, and Osero.

Reviewer: here and elsewhere - make it consistent

```suggestion
summary: Execute the Monthly Settlement Cycle for June 2026 and the associated reconciliation transfers, normalize LSSKY-SKY staking rewards, complete the offboarding of RWA001-A, add emergency spells for STUSDS_MOM to the Chainlog, whitelist the new Osero ALMProxy on the LitePSM, adjust the Osero DC-IAM parameters, whitelist Prime spells for Spark, Grove, and Osero.
```

date: 2026-07-16T00:00:00.000Z
address: "$spell_address"
---

# [Executive Proposal] Monthly Settlement Cycle for June 2026, LSSKY-SKY Rewards Normalization, Complete RWA001-A Offboarding, Add Emergency Spells to the Chainlog, Whitelist Osero ALMProxy, Adjust Vault Parameters, Prime Agent Proxy Spells - July 16, 2026

The Core Facilitators, Dewiz, and Sidestream have placed an executive proposal into the voting system. SKY holders should vote for this proposal if they support the following alterations to the Sky Protocol.

If you are new to voting in the Sky Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.

---

## Executive Summary

If this executive proposal passes, the following **actions** will occur within the Sky Protocol:

- The Monthly Settlement Cycle for June 2026 will be executed.
- LSSKY-SKY staking rewards will be normalized.
- The offboarding of RWA001-A will be completed.

> - Emergency spells relating to the STUSDS_MOM will be added to the [chainlog](https://chainlog.skyeco.com).

Reviewer: no article needed

```suggestion
- Emergency spells relating to STUSDS_MOM will be added to the [chainlog](https://chainlog.skyeco.com).
```

#### Grove

- Mint **12,342,158 USDS** debt in ALLOCATOR-BLOOM-A and transfer the amount to the Surplus Buffer.
- Send **2,328,332 USDS** from the Surplus Buffer to the [GROVE_SUBPROXY](https://etherscan.io/address/0x1369f7b2b38c76B6478c0f0E66D94923421891Ba).
- Send **34,902 USDS** from the Surplus Buffer to the [Demand-Side Buffer](https://etherscan.io/address/0x5e2fEc3a3C4E63A422e45C1BB83EdB3a5aD0543B).

#### Keel

- Send **77,284 USDS** from the Surplus Buffer to the [KEEL_SUBPROXY](https://etherscan.io/address/0x355CD90Ecb1b409Fdf8b64c4473C3B858dA2c310).

#### Obex

- Mint **3,450,783 USDS** debt in ALLOCATOR-OBEX-A and transfer the amount to the Surplus Buffer.
- Send **1,519,539 USDS** from the Surplus Buffer to the [OBEX_SUBPROXY](https://etherscan.io/address/0x8be042581f581E3620e29F213EA8b94afA1C8071).

#### Skybase

- Send **204,242 USDS** from the Surplus Buffer to the [SKYBASE_SUBPROXY](https://etherscan.io/address/0x08978E3700859E476201c1D7438B3427e3C81140).

#### Treasury Management Function

- Transfer **3,378,069 USDS** from the Surplus Buffer to the [Core Council Buffer](https://etherscan.io/address/0x210CFcF53d1f9648C1c4dcaEE677f0Cb06914364).

> This amount represents a **635,130 USDS** Grove Core Council Genesis Capital Expenses Repayment, **1,371,470 USDS** allocated to the Core Council, and **1,371,469 USDS** allocated to the Fortification Conserver, with the Fortification Conserver portion to be paid out by the Core Council Multisig. These items have been combined for coding simplicity and gas efficiency.

Reviewer: ```suggestion
This amount represents a **635,130 USDS** Grove Core Council Genesis Capital Expenses Repayment, **1,371,470 USDS** allocated to the Core Council, and **1,371,469 USDS** allocated to the Fortification Foundation, with the Fortification Foundation portion to be paid out by the Core Council Multisig. These items have been combined for coding simplicity and gas efficiency.
```

This will write off any remaining debt attached to the facility.

### Add STUSDS_MOM Emergency Spells to the Chainlog

- **Authorization**: [Core Facilitator Approval](https://forum.skyeco.com/t/technical-scope-of-the-new-stusdsmom-deploy-and-replacement/27967/7)
- **Proposal**: [Technical Scope](https://forum.skyeco.com/t/technical-scope-of-the-new-stusdsmom-deploy-and-replacement/27967/5)

If this executive proposal passes, then the following standby spell factories and standalone halt spell pertaining to [STUSDS_MOM](https://etherscan.io/address/0x99159d0b885CC6633daC7CD4d82e4247A834b89A) will be added to the [chainlog](https://chainlog.skyeco.com):

- `EMSP_STUSDS_RATE_S_DISS_BUD_FAB`: [**0xb3Fd827F58989cFacFE50d2F8e86A1113b6066D1**](https://etherscan.io/address/0xb3Fd827F58989cFacFE50d2F8e86A1113b6066D1)
- `EMSP_STUSDS_WIPE_PARAM_FAB`: [**0x768D5Ce639c7E7d51E1244E2634d6149bd0d8096**](https://etherscan.io/address/0x768D5Ce639c7E7d51E1244E2634d6149bd0d8096)
- `EMSP_STUSDS_RATE_SETTER_HALT`: [**0x91808ABeCd82495a4a7bf27d80C8c1e89de9effb**](https://etherscan.io/address/0x91808ABeCd82495a4a7bf27d80C8c1e89de9effb)

### Whitelist Osero ALMProxy on LitePSM

> - **Authorization**: [Governance Poll ID 1641](https://vote.sky.money/polling/Qmd3yk9L)

Reviewer: Poll doesn't match shee

```suggestion
- **Authorization**: [Governance Poll ID 1640](https://vote.sky.money/polling/QmUnVyGg)
```

- **Proposal**: [Prime Technical Scope](https://forum.skyeco.com/t/july-16-2026-proposed-changes-to-osero-for-upcoming-spell/28023)

If this executive proposal passes, then the Osero ALMProxy at [`0x6d370e359e9cbd0Fd35Bb38fAF705D84238CB884`](https://etherscan.io/address/0x6d370e359e9cbd0Fd35Bb38fAF705D84238CB884) will be whitelisted on [`MCD_LITE_PSM_USDC_A`](https://etherscan.io/address/0xf6e72Db5454dd049d0788e411b06CfAF16853042).

### Adjust Osero DC-IAM Parameters

- **Authorization**: [Core Council Risk Advisor Approval](https://forum.skyeco.com/t/july-16-2026-proposed-changes-to-osero-for-upcoming-spell/28023/2)
- **Proposal**: [Prime Technical Scope](https://forum.skyeco.com/t/july-16-2026-proposed-changes-to-osero-for-upcoming-spell/28023)

If this executive proposal passes, then the following DC-IAM parameters will be updated for the Osero vault (ALLOCATOR-PRYSM-A):

- Decrease ALLOCATOR-PRYSM-A [Maximum Debt Ceiling (`line`)](https://sky-atlas.io/#6ba18f25-dae8-4fa5-929e-3c7071b70107) by 5 million USDS from 10 million USDS to **5 million USDS**

> - Decrease ALLOCATOR-PRYSM-A [Target Available Debt (`gap`)](https://sky-atlas.io/#07353080-4346-4ffd-bfc8-913cac78776a) by 9 million USDS from 10 million USDS to **1 million USDS**

Reviewer: full sentences should end in periods - see style guide

```suggestion
- Decrease ALLOCATOR-PRYSM-A [Maximum Debt Ceiling (`line`)](https://sky-atlas.io/#6ba18f25-dae8-4fa5-929e-3c7071b70107) by 5 million USDS from 10 million USDS to **5 million USDS**.
- Decrease ALLOCATOR-PRYSM-A [Target Available Debt (`gap`)](https://sky-atlas.io/#07353080-4346-4ffd-bfc8-913cac78776a) by 9 million USDS from 10 million USDS to **1 million USDS**.
```

- `EMSP_STUSDS_WIPE_PARAM_FAB`: [**0x768D5Ce639c7E7d51E1244E2634d6149bd0d8096**](https://etherscan.io/address/0x768D5Ce639c7E7d51E1244E2634d6149bd0d8096)
- `EMSP_STUSDS_RATE_SETTER_HALT`: [**0x91808ABeCd82495a4a7bf27d80C8c1e89de9effb**](https://etherscan.io/address/0x91808ABeCd82495a4a7bf27d80C8c1e89de9effb)

### Prime Agent Proxy Spells

If this executive proposal passes, then a Spark proxy spell with address [`0xC1090e8fEE666868622a2F1e870185F944108Ee2`](https://etherscan.io/address/0xC1090e8fEE666868622a2F1e870185F944108Ee2) and codehash `0xa7b0b6c470169f40016d354a8944f9e5f3f787159fec5192694e8f1ddbc7f26f` will be whitelisted in the [Spark StarGuard](https://etherscan.io/address/0x6605aa120fe8b656482903E7757BaBF56947E45E).

If this executive proposal passes, then a Grove proxy spell with address [`0x4f1318bC0490dC7C7F8230F1dd91A4A2e4694134`](https://etherscan.io/address/0x4f1318bC0490dC7C7F8230F1dd91A4A2e4694134) and codehash `0x38dd6399490f19d7a7e17a6eafda5d2ad92f9395c08efe2fdbacfda3b6d36a89` will be whitelisted in the [Grove StarGuard](https://etherscan.io/address/0xfc51CAa049E8894bEcFfB68c61095C3F3Ec8a880).

> If this executive proposal passes, then an Osero proxy spell with address [`0xd3e4e16ED515Be794fd181D7d2cEB0447A6f2cb5`](https://etherscan.io/address/0xd3e4e16ED515Be794fd181D7d2cEB0447A6f2cb5) and codehash `0x7dca6bc3a3097897198698a674bc824fd3bbb1c9b94c655cd0d78c49db2b9f3e` will be whitelisted in the [Prysm StarGuard](https://etherscan.io/address/0xBfA2D1dA838E55A74c61699e164cDFF8cF0cF0e2).

Reviewer: Osero spell address is wrong - flagged by crafter. Full formal review to follow

```suggestion
If this executive proposal passes, then an Osero proxy spell with address [`0x5D9311fcDda62c08EB9F1115Ca804881a6660445`](https://etherscan.io/address/0x5D9311fcDda62c08EB9F1115Ca804881a6660445) and codehash `0x7dca6bc3a3097897198698a674bc824fd3bbb1c9b94c655cd0d78c49db2b9f3e` will be whitelisted in the [Prysm StarGuard](https://etherscan.io/address/0xBfA2D1dA838E55A74c61699e164cDFF8cF0cF0e2).
```

- Spark Savings (spUSDG) onchain parameters:
  - Deposit cap and max rate unchanged
  - Setter role: [0xAEa9f5dE56e6C20383a1fcC2C3629Dca0A92cE41](https://robinhoodchain.blockscout.com/address/0xAEa9f5dE56e6C20383a1fcC2C3629Dca0A92cE41) (ALM Proxy Freezable)
- SLL rate limits
  - transferAsset (USDG)
    - recipient: [0x17C0F5345d1144fdF670D14719077be3842E5087](https://robinhoodchain.blockscout.com/address/0x17C0F5345d1144fdF670D14719077be3842E5087)
    - maxAmount: **50,000,000 USDG**
    - slope: **250,000,000 USDG per day**

##### [Ethereum] Spark Liquidity Layer - Enable USDG Bridging to Robinhood Chain

- **Authorization**: [Snapshot Poll](https://snapshot.box/#/s:sparkfi.eth/proposal/0xd177bc28b65afb23dc39a5e7cfdded7084b3b722b230e08d7067b68fa0f4486a)
- **Proposal**: [Prime Technical Scope](https://forum.skyeco.com/t/july-16-2026-proposed-changes-to-spark-for-upcoming-spell/28029)

If this executive proposal passes, then the Spark proxy spell will enable bridging of USDG from Ethereum to Robinhood Chain with the following [rate limits](https://sky-atlas.io/#8efb0a11-b798-48eb-af19-f65b38f039b5):

- transferAsset (USDG)

>   - recipient: [0xf752cF318dfF2C01575c98741AA52e7a34d873Fd](https://etherscan.io/address/0x17C0F5345d1144fdF670D14719077be3842E5087)

Reviewer: Incorrect link

```suggestion
  - recipient: [0xf752cF318dfF2C01575c98741AA52e7a34d873Fd](https://etherscan.io/address/0xf752cF318dfF2C01575c98741AA52e7a34d873Fd)
```

  - destination: [0xfD2fD4B046136B540A56C11c75ac679AE7d1dB24](https://robinhoodchain.blockscout.com/address/0xfD2fD4B046136B540A56C11c75ac679AE7d1dB24)
  - maxAmount: **50,000,000 USDG**

>   - slope: **250,000,000 USDG per day**

Reviewer: spell large numbers per style guide

```suggestion
  - maxAmount: **50 million USDG**
  - slope: **250 million USDG per day**
```

  - Freezer: [0x90D8c80C028B4C09C0d8dcAab9bbB057F0513431](https://web3.okx.com/explorer/x-layer/evm/address/0x90D8c80C028B4C09C0d8dcAab9bbB057F0513431)
  - Default admin: [0xCF5af6F53ceC74B791cb4182aC778ca9CD323510](https://web3.okx.com/explorer/x-layer/evm/address/0xCF5af6F53ceC74B791cb4182aC778ca9CD323510) (Spark Executor on X Layer)

##### [Ethereum] Spark Liquidity Layer - Enable USDT Bridging to X Layer

- **Authorization**: [Snapshot Poll](https://snapshot.box/#/s:sparkfi.eth/proposal/0xdde478db4ba5882a5d48d19fdbae057fd703688e4f1e16fb673407fc08476a9f)
- **Proposal**: [Prime Technical Scope](https://forum.skyeco.com/t/july-16-2026-proposed-changes-to-spark-for-upcoming-spell/28029)

If this executive proposal passes, then the Spark proxy spell will enable USDT bridging to X Layer with the following [rate limits](https://sky-atlas.io/#8efb0a11-b798-48eb-af19-f65b38f039b5):

- Bridge USDT to X Layer (Ethereum → X Layer route)
  - destination: [0x83A914C361bB729EB6BEBC8C7bA993667A0E6Df8](https://web3.okx.com/explorer/x-layer/evm/address/0x83A914C361bB729EB6BEBC8C7bA993667A0E6Df8) (ALM Proxy on X Layer)
  - maxAmount: **5,000,000 USDT**

>   - slope: **100,000,000 USDT per day**

Reviewer: ```suggestion
  - maxAmount: **5 million USDT**
  - slope: **100 million USDT per day**
```

- **Authorization**: [Snapshot Poll](https://snapshot.box/#/s:sparkfi.eth/proposal/0xf99372ccca4b99dd04dc0ddb038e949b62f4d25810b0203572dc90bce025e805)
- **Proposal**: [Prime Technical Scope](https://forum.skyeco.com/t/july-16-2026-proposed-changes-to-spark-for-upcoming-spell/28029)

If this executive proposal passes, then the Spark proxy spell will remove the old, non-frontend-listed USDT Morpho V2 vault from the Spark Liquidity Layer by updating the following [rate limits](https://sky-atlas.io/#8efb0a11-b798-48eb-af19-f65b38f039b5):

- Vault address: [0xc7CDcFDEfC64631ED6799C95e3b110cd42F2bD22](https://etherscan.io/address/0xc7CDcFDEfC64631ED6799C95e3b110cd42F2bD22)
- Deposit and withdrawal rate limits:
  - maxAmount: **0 USDT**
  - slope: **0 USDT per day**

##### [Ethereum] Spark Treasury - USDS Transfer to Spark Foundation for Incentives

- **Authorization**: [Snapshot Poll](https://snapshot.box/#/s:sparkfi.eth/proposal/0x2df281a276e0c17eff9a05e65bfc05937c2f600edec1a82386a6efb6dbe9d63d)
- **Proposal**: [Prime Technical Scope](https://forum.skyeco.com/t/july-16-2026-proposed-changes-to-spark-for-upcoming-spell/28029)

> If this executive proposal passes, then the Spark proxy spell will transfer **2 million USDS** from the Spark SubProxy to the Spark Foundation at [0x92e4629a4510AF5819d7D1601464C233599fF5ec](https://etherscan.io/address/0x92e4629a4510AF5819d7D1601464C233599fF5ec) to fund exchange/distributor hold-to-earn initiatives and term deposit boost programs.

Reviewer: ```suggestion
If this executive proposal passes, then the Spark proxy spell will transfer **2 million USDS** from the Spark SubProxy to the Spark Foundation at [0x92e4629a4510AF5819d7D1601464C233599fF5ec](https://etherscan.io/address/0x92e4629a4510AF5819d7D1601464C233599fF5ec) to fund exchange and distributor hold-to-earn initiatives and term deposit boost programs.
```

##### [Ethereum] Spark Treasury - USDS Transfer to Spark Assets Foundation for Anchorage Fees

- **Authorization**: [Snapshot Poll](https://snapshot.box/#/s:sparkfi.eth/proposal/0x3cb4165f1357d553445b0de790e4e8b4a71358f42f39d35f7de51b308ade558c)
- **Proposal**: [Prime Technical Scope](https://forum.skyeco.com/t/july-16-2026-proposed-changes-to-spark-for-upcoming-spell/28029)

If this executive proposal passes, then the Spark proxy spell will transfer **500,000 USDS** from the Spark SubProxy to the Spark Assets Foundation at [0xEabCb8C0346Ac072437362f1692706BA5768A911](https://etherscan.io/address/0xEabCb8C0346Ac072437362f1692706BA5768A911) to cover Anchorage variable fees.

##### [Ethereum] Spark Liquidity Layer - Transfer USDS to Grove

- **Authorization**: [Snapshot Poll](https://snapshot.box/#/s:sparkfi.eth/proposal/0x451bb53f80ad2906ff06cc3d03c88a6b09f350db371c4782e0621e26a1d55a43)
- **Proposal**: [Prime Technical Scope](https://forum.skyeco.com/t/july-16-2026-proposed-changes-to-spark-for-upcoming-spell/28029)

> If this executive proposal passes, then the Spark proxy spell will transfer an amount of USDS to the Grove ALMProxy at [0x491EDFB0B8b608044e227225C715981a30F3A44E](https://etherscan.io/address/0x491EDFB0B8b608044e227225C715981a30F3A44E) in exchange for the Grove ALMProxy's balance of syrupUSDC. The precise amount of USDS will be determined by the formula: `85,943,747.637271 * syrupUSDC/USDC conversion rate` at the time of execution, in USDS.

Reviewer: don't need to specify in USDS as it is clear from the rest of the sentence

```suggestion
If this executive proposal passes, then the Spark proxy spell will transfer an amount of USDS to the Grove ALMProxy at [0x491EDFB0B8b608044e227225C715981a30F3A44E](https://etherscan.io/address/0x491EDFB0B8b608044e227225C715981a30F3A44E) in exchange for the Grove ALMProxy's balance of syrupUSDC. The precise amount of USDS will be determined by the formula: `85,943,747.637271 * syrupUSDC/USDC conversion rate` at the time of execution.
```

##### [Ethereum] Spark Treasury - Transfer USDS for Buybacks

- **Authorization**: [Sky Atlas](https://sky-atlas.io/#6a4870fa-73f1-4d49-b7ee-d531fb59a971)
- **Proposal**: [Prime Technical Scope](https://forum.skyeco.com/t/july-16-2026-proposed-changes-to-spark-for-upcoming-spell/28029)

If this executive proposal passes, then the Spark proxy spell will transfer **64,231 USDS** to the Spark Operations Multisig at [0x2E1b01adABB8D4981863394bEa23a1263CBaeDfC](https://etherscan.io/address/0x2E1b01adABB8D4981863394bEa23a1263CBaeDfC) to fund SPK buybacks.

#### Grove Proxy Spell

The Pull Request for the Grove proxy spell can be viewed [here](https://github.com/grove-labs/grove-spells/pull/63).

##### [Ethereum + Robinhood Chain] Onboard the Grove Liquidity Layer on Robinhood Chain

- **Authorization**: [Governance Poll ID 1640](https://vote.sky.money/polling/QmUnVyGg)
- **Proposal**: [Prime Technical Scope](https://forum.skyeco.com/t/july-16-2026-proposed-changes-to-grove-for-upcoming-spell/28024)

> If this executive proposal passes, then the Grove proxy spell will onboard the Grove Liquidity Layer to Robinhood chain with the following contracts and rate limits.

Reviewer: 1. Robinhood Chain should be capitalized
2. add rate limits link

```suggestion
If this executive proposal passes, then the Grove proxy spell will onboard the Grove Liquidity Layer to Robinhood Chain with the following contracts and [rate limits](https://sky-atlas.io/#8efb0a11-b798-48eb-af19-f65b38f039b5):
```

| Robinhood Chain Gov-Relay Receiver (Arbitrum-native `ArbitrumReceiver`) | [`rh:0xa02eC279eEA9E56F4E14449a07C5ca5FDAAdc51d`](https://robinhoodchain.blockscout.com/address/0xa02eC279eEA9E56F4E14449a07C5ca5FDAAdc51d) |
| Robinhood Chain Delayed Inbox (Ethereum Mainnet — Item 1) | [`eth:0x1A07cc4BD17E0118BdB54D70990D2158AbAD7a2D`](https://etherscan.io/address/0x1A07cc4BD17E0118BdB54D70990D2158AbAD7a2D) |
| USDG (Robinhood Chain) | [`rh:0x5fc5360D0400a0Fd4f2af552ADD042D716F1d168`](https://robinhoodchain.blockscout.com/address/0x5fc5360D0400a0Fd4f2af552ADD042D716F1d168) |
| Paxos USDG receive address (Robinhood Chain) | [`rh:0xfC0a7Ed7C5146B26eB38FA92c71F434A7178b06e`](https://robinhoodchain.blockscout.com/address/0xfC0a7Ed7C5146B26eB38FA92c71F434A7178b06e) |

###### Initial Parameters

[Rate limits](https://sky-atlas.io/#8efb0a11-b798-48eb-af19-f65b38f039b5) for initial activation:

| Parameter | Max Amount | Slope (Tokens/Day) |
| --- | --- | --- |
| LIMIT_ASSET_TRANSFER (USDC from mainnet ALMProxy to Paxos USDC receive address ) | **50,000,000 USDC** | **50,000,000 USDC/day** |

> | LIMIT_ASSET_TRANSFER (USDG from Robinhood Chain ALMProxy to Paxos USDG receive address) | **50,000,000 USDG** | **50,000,000 USDG/day** |

Reviewer: ```suggestion
| LIMIT_ASSET_TRANSFER (USDC from mainnet ALMProxy to Paxos USDC receive address ) | **50 million USDC** | **50 million USDC/day** |
| LIMIT_ASSET_TRANSFER (USDG from Robinhood Chain ALMProxy to Paxos USDG receive address) | **50 million USDG** | **50 million USDG/day** |
```

These contracts and rate limits will activate the Grove Liquidity Layer on Robinhood Chain and activate bridging of USDG in both directions. The conversion from USDC to USDG is an off-chain conversion handled by Paxos.

> ##### [Ethereum] Onboard the Robinhood Chain Grove x Steakhouse USDG Morpho Vault for Grove ALM deposits

Reviewer: ```suggestion
##### [Robinhood Chain] Onboard the Robinhood Chain Grove x Steakhouse USDG Morpho Vault for Grove ALM deposits
```

- **Authorization**: [Governance Poll ID 1640](https://vote.sky.money/polling/QmUnVyGg)
- **Proposal**: [Prime Technical Scope](https://forum.skyeco.com/t/july-16-2026-proposed-changes-to-grove-for-upcoming-spell/28024)

If this executive proposal passes, then the Grove proxy spell will onboard the Robinhood Chain Grove x Steakhouse USDG Morpho vault with the following [rate limits](https://sky-atlas.io/#8efb0a11-b798-48eb-af19-f65b38f039b5):

- Vault address: [0xBEEff039907422219Fb367e525954DDC092854d9](https://robinhoodchain.blockscout.com/address/0xBEEff039907422219Fb367e525954DDC092854d9)
- Deposits:
  - maxAmount: **50,000,000 USDG**

>   - slope: **50,000,000 USDG/day**

Reviewer: ```suggestion
  - maxAmount: **50 million USDG**
  - slope: **50 million USDG/day**
```

| Core Operator Relayer / Actor (Soter Labs) | [`0x3dE688267Cf099307aBdd85F64D8efe03D0b2b26`](https://etherscan.io/address/0x3dE688267Cf099307aBdd85F64D8efe03D0b2b26) |
| Freezer / Revoker | [`0xF61F90907551a8A23f0f8EEE9658Fa53326de603`](https://etherscan.io/address/0xF61F90907551a8A23f0f8EEE9658Fa53326de603) |

The proxy spell will:

- Set the controller's USDS vault: point the Diamond PAU Controller at the `ALLOCATOR-PRYSM-A` AllocatorVault.
- Authorize the DPAU ALMProxy on the AllocatorVault.
- Approve the DPAU ALMProxy on the AllocatorBuffer.

###### Initial Parameters

[Rate limits](https://sky-atlas.io/#8efb0a11-b798-48eb-af19-f65b38f039b5) for initial activation:

| Parameter | Max Amount | Slope (Tokens/Day) |
| --- | --- | --- |
| LIMIT_USDS_MINT | **5 million USDS** | **5 million USDS/day** |
| LIMIT_USDS_BURN | **5 million USDS** | **5 million USDS/day** |

##### [Ethereum] Onboard SparkLend USDS (spUSDS)

- **Authorization**: [Governance Poll ID 1640](https://vote.sky.money/polling/QmUnVyGg)
- **Proposal**: [Prime Technical Scope](https://forum.skyeco.com/t/july-16-2026-proposed-changes-to-osero-for-upcoming-spell/28023)

> If this executive proposal passes, then  the Osero proxy spell will onboard SparkLend USDS (spUSDS) with the following [rate limits](https://sky-atlas.io/#8efb0a11-b798-48eb-af19-f65b38f039b5):

Reviewer: ```suggestion
If this executive proposal passes, then the Osero proxy spell will onboard SparkLend USDS (spUSDS) with the following [rate limits](https://sky-atlas.io/#8efb0a11-b798-48eb-af19-f65b38f039b5):
```

---
title: Template - [Executive Vote] Monthly Settlement Cycle for June 2026, LSSKY-SKY Rewards Normalization, Complete RWA001-A Offboarding, Add Emergency Spells to the Chainlog, Whitelist Osero ALMProxy, Adjust Vault Parameters, Update Safe Harbor Agreement, Prime Agent Proxy Spells - July 16, 2026

> summary: Execute the Monthly Settlement Cycle for June 2026 and the associated reconciliation transfers, normalize LSSKY-SKY staking rewards, complete the offboarding of RWA001-A, add emergency spells for STUSDS_MOM to the Chainlog, whitelist the new Osero ALMProxy on the LitePSM, adjust the Osero DC-IAM parameters, update the Safe Harbor Agreement, whitelist Prime spells for Spark, Grove, and Osero.

Reviewer 2: ```suggestion
summary: Execute the Monthly Settlement Cycle for June 2026 and the associated reconciliation transfers, normalize LSSKY-SKY staking rewards, complete the offboarding of RWA001-A, add emergency spells for STUSDS_MOM to the Chainlog, whitelist the new Osero ALMProxy on the LitePSM, adjust the Osero DC-IAM parameters, update the Safe Harbor Agreement, whitelist Prime Agent spells for Spark, Grove, and Osero.
```

date: 2026-07-16T00:00:00.000Z
address: "$spell_address"
---

# [Executive Proposal] Monthly Settlement Cycle for June 2026, LSSKY-SKY Rewards Normalization, Complete RWA001-A Offboarding, Add Emergency Spells to the Chainlog, Whitelist Osero ALMProxy, Adjust Vault Parameters, Update Safe Harbor Agreement, Prime Agent Proxy Spells - July 16, 2026

The Core Facilitators, Dewiz, and Sidestream have placed an executive proposal into the voting system. SKY holders should vote for this proposal if they support the following alterations to the Sky Protocol.

If you are new to voting in the Sky Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.

---

## Executive Summary

If this executive proposal passes, the following **actions** will occur within the Sky Protocol:

- The Monthly Settlement Cycle for June 2026 will be executed.

> - LSSKY-SKY staking rewards will be normalized.

Reviewer 2: ```suggestion
- LSSKY->SKY staking rewards will be normalized.
```

If this executive proposal passes, then the Osero ALMProxy at [`0x6d370e359e9cbd0Fd35Bb38fAF705D84238CB884`](https://etherscan.io/address/0x6d370e359e9cbd0Fd35Bb38fAF705D84238CB884) will be whitelisted on [`MCD_LITE_PSM_USDC_A`](https://etherscan.io/address/0xf6e72Db5454dd049d0788e411b06CfAF16853042).

### Adjust Osero DC-IAM Parameters

- **Authorization**: [Core Council Risk Advisor Approval](https://forum.skyeco.com/t/july-16-2026-proposed-changes-to-osero-for-upcoming-spell/28023/2)
- **Proposal**: [Prime Technical Scope](https://forum.skyeco.com/t/july-16-2026-proposed-changes-to-osero-for-upcoming-spell/28023)

If this executive proposal passes, then the following DC-IAM parameters will be updated for the Osero vault (ALLOCATOR-PRYSM-A):

- Decrease ALLOCATOR-PRYSM-A [Maximum Debt Ceiling (`line`)](https://sky-atlas.io/#6ba18f25-dae8-4fa5-929e-3c7071b70107) by 5 million USDS from 10 million USDS to **5 million USDS**.
- Decrease ALLOCATOR-PRYSM-A [Target Available Debt (`gap`)](https://sky-atlas.io/#07353080-4346-4ffd-bfc8-913cac78776a) by 9 million USDS from 10 million USDS to **1 million USDS**.

> - Leave ALLOCATOR-PRYSM-A [Stability Fee (`duty`)](https://sky-atlas.io/#0257a420-e92e-4942-b794-a559f299365f) and [Ceiling Increase Cooldown (`ttl`)](https://sky-atlas.io/#a5ae79ad-9460-41a3-8dbf-65605f54b79b) unchanged.

Reviewer 2: Suggestion: Include the figures for `duty` and `ttl`.

We now have the ttl figure in the sheet, however since the parameters are unchanged I am okay with leaving as is.

| Robinhood Chain Executor | [`rh:0x5ff98717a18833de1A49e11B498866d6Fa1c9296`](https://robinhoodchain.blockscout.com/address/0x5ff98717a18833de1A49e11B498866d6Fa1c9296) |
| Robinhood Chain Gov-Relay Receiver (Arbitrum-native `ArbitrumReceiver`) | [`rh:0xa02eC279eEA9E56F4E14449a07C5ca5FDAAdc51d`](https://robinhoodchain.blockscout.com/address/0xa02eC279eEA9E56F4E14449a07C5ca5FDAAdc51d) |
| Robinhood Chain Delayed Inbox (Ethereum Mainnet — Item 1) | [`eth:0x1A07cc4BD17E0118BdB54D70990D2158AbAD7a2D`](https://etherscan.io/address/0x1A07cc4BD17E0118BdB54D70990D2158AbAD7a2D) |
| USDG (Robinhood Chain) | [`rh:0x5fc5360D0400a0Fd4f2af552ADD042D716F1d168`](https://robinhoodchain.blockscout.com/address/0x5fc5360D0400a0Fd4f2af552ADD042D716F1d168) |
| Paxos USDG receive address (Robinhood Chain) | [`rh:0xfC0a7Ed7C5146B26eB38FA92c71F434A7178b06e`](https://robinhoodchain.blockscout.com/address/0xfC0a7Ed7C5146B26eB38FA92c71F434A7178b06e) |

###### Initial Parameters

[Rate limits](https://sky-atlas.io/#8efb0a11-b798-48eb-af19-f65b38f039b5) for initial activation:

| Parameter | Max Amount | Slope (Tokens/Day) |
| --- | --- | --- |

> | LIMIT_ASSET_TRANSFER (USDC from mainnet ALMProxy to Paxos USDC receive address ) | **50 million USDC** | **50 million USDC/day** |

Reviewer 2: ```suggestion
| LIMIT_ASSET_TRANSFER (USDC from mainnet ALMProxy to Paxos USDC receive address) | **50 million USDC** | **50 million USDC/day** |
```

| LIMIT_ASSET_TRANSFER (USDG from Robinhood Chain ALMProxy to Paxos USDG receive address) | **50 million USDG** | **50 million USDG/day** |

These contracts and rate limits will activate the Grove Liquidity Layer on Robinhood Chain and activate bridging of USDG in both directions. The conversion from USDC to USDG is an off-chain conversion handled by Paxos.

> ##### [Robinhood Chain] Onboard the Robinhood Chain Grove x Steakhouse USDG Morpho Vault for Grove ALM deposits

Reviewer: ```suggestion
##### [Robinhood Chain] Onboard the Robinhood Chain Grove x Steakhouse USDG Morpho Vault for Grove ALM Deposits
```
